components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/utils/CmdUtils.java

```diff
@@ -228,6 +228,32 @@ public static String getCLIHome() {
         return System.getProperty(CliConstants.CLI_HOME);
     }
 
+    /**
+     * Get cacerts location
+     *
+     * @return cacerts location
+     */
+    public static String getCacertsLocation() {
+        return System.getProperty(CliConstants.CACERTS_DIR);
+    }
+
+    /**
+     * Get cacerts password
+     *
+     * @return cacerts password
+     */
+    public static String getCacertsPassword() {
+        String cacertsPassword = "";
+        String password = System.getenv("CACERTS_PASS");
+
+        if (password != null) {
+            cacertsPassword = password;
+        } else {
+            cacertsPassword = CliConstants.DEFAULT_CACERTS_PASS;
+        }
```

Comment:
Shall we make this logic one line?

```suggestion
        cacertsPassword = password != null ? password : CliConstants.DEFAULT_CACERTS_PASS;
```